### ISSUE TITLE
[FEATURE] A way to configure @html-eslint/attrs-newline closeStyle only

### ISSUE THREAD

**Issue by jpradelle** [yeonjuan/html-eslint#593]
I would like to configure `@html-eslint/attrs-newline` with `"closeStyle": "sameline"` but without restriction on attributes number allowed for single line tag.

I tried to configure like that, to avoid constraints on attributes number I put a large number
```json
{
  "rules": {
    "@html-eslint/attrs-newline": ["error", {
      "closeStyle": "sameline",
      "ifAttrsMoreThan": 10
    }]
  }
}
```
But if number of attributes is not reached, `closeStyle` rule is not applied.

On this playground: no issue is reported about close style. If you change `ifAttrsMoreThan` to 2, then close style issue get reported.
https://html-eslint.org/playground#eyJjb2RlIjoiPCFkb2N0eXBlIGh0bWw+XG4gICAgPGh0bWwgbGFuZz1cImVuLVVTXCI+XG4gICAgICA8Ym9keT5cbiAgICAgICAgPGZvby1iYXJcbiAgICAgICAgICAgIGF0dHIxPVwiYVwiXG4gICAgICAgICAgICBhdHRyMj1cImJcIlxuICAgICAgICAgICAgYXR0cjM9XCJjXCI+PC9mb28tYmFyPlxuICAgICAgICA8Zm9vLWJhclxuICAgICAgICAgICAgYXR0cjE9XCJhXCJcbiAgICAgICAgICAgIGF0dHIyPVwiYlwiXG4gICAgICAgICAgICBhdHRyMz1cImNcIlxuICAgICAgICA+PC9mb28tYmFyPlxuICAgICAgPC9ib2R5PlxuICAgIDwvaHRtbD5cbiAgICAiLCJjb25maWciOnsicnVsZXMiOnsiQGh0bWwtZXNsaW50L2F0dHJzLW5ld2xpbmUiOlsiZXJyb3IiLHsiY2xvc2VTdHlsZSI6InNhbWVsaW5lIiwiaWZBdHRyc01vcmVUaGFuIjoxMH1dfX0sImxhbmd1YWdlIjoiaHRtbCJ9

Maybe having a dedicated rule to close style would be nice, or a way to have it applied from this rule, but without restriction on attributes.

I'm not using linting based on number of attributes. But it could be nice to lint attribute chop down, but based based on line length whatever the number of attribute: in my case, if line length is larger than 150 chars, it is requested to chop down attributes.
Maybe this could be a dedicated feature request ?
If you use IntelliJ, this correspond to setup Editor > Code Style > HTML > Other:  `Wrap attributes: chop down if long` and `New line before first attribute: when multiline` and Editor > Code Style: `Hard wrap at: 150` (or whatever value)

**Comment by yeonjuan** [comment]
Hi @jpradelle Thanks for the suggestion.

> Maybe having a dedicated rule to close style would be nice, or a way to have it applied from this rule, but without restriction on attributes.

 Hmm, yeah, I think a separate rule might be more appropriate here.
I think it would be best to deprecate the `closeStyle` option of `attrs-newline` in the future.

> I'm not using linting based on number of attributes. But it could be nice to lint attribute chop down, but based based on line length whatever the number of attribute: in my case, if line length is larger than 150 chars, it is requested to chop down attributes.

It might be a good idea to add an option like `maxLen` to `attrs-newline`.
